documentation/src/docs/asciidoc/writing-tests.adoc

```diff
@@ -271,8 +271,74 @@ class MyMockitoTest {
 [[writing-tests-interface-default-methods]]
 === Interface Default Methods
 
-JUnit Jupiter allows `@Test`, `@TestFactory`, `@BeforeEach`, and `@AfterEach` to be declared on
-interface default methods. One possible application of this feature is to write tests for
+JUnit Jupiter allows `@Test`, `@TestFactory`, `@BeforeEach` and `@AfterEach` to be declared on
```

Comment:
Please don't switch punctuation rules from American English to British English. In other words, please keep the comma that you deleted.